### DISCUSSION TITLE
Hono support?

### DISCUSSION THREAD

**Discussion by empz** [logto-io/logto#6767] [Ideas]
I'd love to see an integration for [Hono](https://hono.dev/) as a simple middleware much like the Express integration.

**Comment by Nu11ified** [comment]
YES! HONO INTERGRATION PLEASE!

**Comment by darcyYe** [comment]
Hi @empz , would you like to post this as a feature request in [issues](https://github.com/logto-io/logto/issues) so that we can better collect users feedbacks?

**Reply by empz** [reply]
Done.

https://github.com/logto-io/logto/issues/6771

**Comment by eehP** [comment]
Any news on it ?
